### PR TITLE
Task #2  added solution in submission folder

### PR DESCRIPTION
**Pull request by PratyushPareek**
__3__ 

#### Short description of what this resolves:
Added the solution (python script) in submissions folder in my name's folder 

#### Changes proposed in this pull request and/or Screenshots of changes:
task2_1
task2_2

-
-
-

### CODE REVIEW

File: Submissions/PratyushPareek/task2.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import shutil
+import os
+
+ 
+for folders, subfolders ,files in os.walk("..\\...\\Monty_Python_Class\\Task_2\\My_Files\\'):
```
Comment (Lokesh2703) [inline Submissions/PratyushPareek/task2.py]: Use os.path.join()

Comment (PratyushPareek) [reply Submissions/PratyushPareek/task2.py]: @Lokesh2703  done 
I've also made a separate branch and a separate PR for task 3

**Comment by Lokesh2703** [comment]
put task2.py In its own folder of task 2

**Comment by PratyushPareek** [comment]
@Lokesh2703  done

**Comment by Lokesh2703** [comment]
Look at the instructions properly and then the file.
Instructions are in readme.md file

**Pull request closed without merging**